constructor/osxpkg.py

```diff
@@ -132,7 +134,8 @@ def modify_xml(xml_path, info):
                 f.write(info["conclusion_text"])
     else:
         conclusion_path = join(OSX_DIR, 'acloud.rtf')
-
+        if "conclusion_file" in info and info["conclusion_file"].endswith(".nsi"):
+            print(f"NSI conclusion_file, {info['conclusion_file'].endswith('.nsi')}, is ignored.")
```

Comment:
```suggestion
            print(f"Warning: NSI conclusion_file, {info['conclusion_file'].endswith('.nsi')}, is ignored.")
```

[Other parts of the code](https://github.com/conda/constructor/blob/02ff321671d791f05978f1ab12e82a0b4b90688d/constructor/utils.py#L145) prefix the message with `Warning: `, adding it here for consistency.

You can also apply the `dict.get(key, "")` trick here if you want!